rules:
  - type: dynamic_paramter
    node_name: 'calculate_odom_faulty_imu_calib'
    paramter_name: 'use_imu'
    paramter_value: 'false'
    positive_possible_faulty_resources:
        - resource: 'imu'
